src/middleware/error-handler.js

```diff
@@ -1,6 +1,6 @@
 const { StatusCodes } = require('http-status-codes');
 
-const errorHandlerMiddleware = (err, req, res, next) => {
+const errorHandlerMiddleware = (err, req, res, _next) => {
```

Comment:
Marking unused arguments with an underscore prefix; tells ESLint to ignore them / that we are intentionally not using that argument.